app/controllers/online_applications_controller.rb

```diff
@@ -45,6 +42,18 @@ def process_application(application)
     ResolverService.new(application, current_user).complete
   end
 
+  def authorise_online_application
```

Comment:
I struggled with that, in the other controllers it's authori**_s_**e, in the gem it's authori**_z_**e.

Maybe in a future refactoring, I'll make them all with Z's